SoftLayer/CLI/firewall/monitoring.py

```diff
@@ -0,0 +1,51 @@
+"""Get monitoring for a firewall device."""
+# :license: MIT, see LICENSE for more details.
+import datetime
+
+import click
+
+import SoftLayer
+from SoftLayer.CLI import environment
+from SoftLayer.CLI import formatting
+
+
+@click.command(cls=SoftLayer.CLI.command.SLCommand, )
+@click.argument('identifier')
+@environment.pass_env
+def cli(env, identifier):
+    """Get details for a firewall monitors device."""
```

Comment:
Update this description to be more specific of what it is actually doing
```suggestion
    """Gets bandwidth details for a firewall from the past 30 days"""
```

Allowing for a custom date range would be useful, but that can be added in a future issue